### DISCUSSION TITLE
Demo reply "invalid_api_key"

### DISCUSSION THREAD

**Discussion by ptmicky** [ddiu8081/chatgpt-demo#417] [Q&A] [answered]
API key is right, firewall is closed, but Demo reply "invalid_api_key", I don't what happened, is there anyone how to slove this?

**Comment by yzh990918** [comment]
Check that the environment variables are configured and then retrigger the deployment.

**Reply by ptmicky** [reply]
I creat a plugin directory by 
`mkdir -p /etc/systemd/system/docker.service.d`
creat a file by
`vim /etc/systemd/system/docker.service.d/http-proxy.conf`
add
`[Service]
Environment="HTTP_PROXY=http://127.0.0.1:7890/"`

or I set the environment by `export OPENAI_API_KEY="[OpenAI API key]" `in centos7
and reload conf file, final restart docker, but there is still not work, any more help would be appreciated

**Reply by ptmicky** [reply, marked as answer]
After set the environment by `export OPENAI_API_KEY="[OpenAI API key]" `in centos7,
and remove docker image by
`docker images`
`docker rmi ace34d1234asdf`
and remove docker by 
`docker ps -a`
`docker rm sadfasdfasccadsf`, then I rearrange the demo in vps centos7, my issue has sloved, and the new issue: "You exceeded your current quota, please check your plan and billing details" is also resolved.
